models/Asset/Image.php

```diff
@@ -404,7 +404,8 @@ private function isAnimatedPng(): bool
              *
              * @see http://foone.org/apng/
              */
-            $isAnimated = str_contains(substr($fileContent, 0, strpos($fileContent, 'IDAT')), 'acTL');
+            $posIDAT = strpos($fileContent, 'IDAT');
+            $isAnimated = $posIDAT !== false ? strpos(substr($fileContent, 0, $posIDAT), 'acTL') !== false : false;
```

Comment:
Why did you change `str_contains()` to `strpos()`?

```suggestion
            $isAnimated = $posIDAT !== false ? str_contains(substr($fileContent, 0, $posIDAT), 'acTL') : false;
```

Reply:
There was strpos in pimcore 11.0.3.
![image](https://github.com/pimcore/pimcore/assets/30948231/c255cbe3-306b-4db0-b1f7-cb06a2be9158)